src/main/java/mat/server/service/impl/LoginCredentialServiceImpl.java

```diff
@@ -330,6 +328,23 @@ public boolean isValidPassword(String userId, String password) {
         }
     }
 
+    public LoginModel initSession(String harpId, String sessionId, String accessToken) {
+        logger.debug("setUpUserSession::" + harpId + "::" + sessionId);
+        // Set Session ID in user details.
+        MatUserDetails userDetails = (MatUserDetails) hibernateUserService.loadUserByHarpId(harpId);
+        if(userDetails == null) {
+            throw new IllegalArgumentException("HARP_ID_NOT_FOUND");

```

Comment:
I have a new interface to store HarpConstants. We can use it for Harp related messages and Constants.